### DISCUSSION TITLE
How can i create a new block ?

### DISCUSSION THREAD

**Discussion by censonalvin** [GrapesJS/grapesjs#4508] [Q&A] [answered]
Hi guys. I'm new at using grapesJS. I was trying to seperate each of this blocks/categories (see the pic) and call it on different divs on the sidenav. How can i do that? I can't seem to find a tutorial/s or a documentation regarding on what i'm trying to do. (might lack exploring on my side).

![image](https://user-images.githubusercontent.com/17976518/184295835-32ebff8b-e28c-4508-a016-02444bf6eaba.png)

**Comment by artf** [comment, marked as answer]
Try to filter them via render method https://grapesjs.com/docs/api/block_manager.html#render

**Reply by censonalvin** [reply]
where do i need to place it ? when i'm trying to place it on the last line of my JS. i'm getting "Uncaught ReferenceError: blockManager is not defined".

**Reply by artf** [reply]
```js
const blockManager = editor.Blocks;
blockManager.render...
```

**Reply by censonalvin** [reply]
Just wanna ask if the 'some-id' is where the block will should go right? (reference below).

-> document.getElementById('some-id').appendChild(newBlocksEl);

**Reply by censonalvin** [reply]
btw its working thanks. now i will gonna find out how to include the blocks that we made to each categories. it seem it is only getting the ones in the plugin. 

Thanks again @artf
